lineman/app/components/utilities.scss

```diff
@@ -154,6 +154,18 @@ label.label-with-details{
   margin-top: 22px;
 }
 
+.lmo-card {
```

Comment:
@AaronThornton00 I've removed all the padding declarations in favor of using lmo-card and lmo-card--no-padding.